src/main/java/com/stripe/model/Account.java

```diff
@@ -598,25 +604,28 @@ public CapabilityCollection capabilities(AccountCapabilitiesParams params, Reque
   public static class Capabilities extends StripeObject {
     /**
      * The status of the card issuing capability of the account, or whether you can use Issuing to
-     * distribute funds on cards.
+     * distribute funds on cards One of `active`, `inactive`, or `pending`.
```

Comment:
Looks like the original documentation string is lacking a final `.`. Another reason to prefix the enum string with `<p>`.